<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsRelationManagerAddDialogBase</class>
 <widget class="QWidget" name="QgsRelationManagerAddDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Relation</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="1,2,0">
   <item row="3" column="0">
    <widget class="QLabel" name="mRelationStrengthLabel">
     <property name="text">
      <string>Relationship strength</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QTableWidget" name="mFieldsMappingTable">
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="rowCount">
      <number>1</number>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>150</number>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <row>
      <property name="text">
       <string>Layer</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string comment="static relations">Referenced (parent)</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Referencing (child)</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="mIdLabel">
     <property name="text">
      <string>Id</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="mNameLabel">
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QComboBox" name="mRelationStrengthComboBox"/>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLineEdit" name="mIdLineEdit">
     <property name="placeholderText">
      <string>[Generated automatically]</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="mButtonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QLineEdit" name="mNameLineEdit"/>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QLabel" name="mFieldsMappingLabel">
     <property name="text">
      <string>Layer and fields mapping</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <layout class="QVBoxLayout" name="mFieldsMappingButtonsVBoxLayout">
     <item>
      <widget class="QToolButton" name="mFieldsMappingAddButton">
       <property name="toolTip">
        <string>Add new field pair as part of a composite foreign key</string>
       </property>
       <property name="icon">
        <iconset resource="../../images/images.qrc">
         <normaloff>:/images/themes/default/symbologyAdd.svg</normaloff>:/images/themes/default/symbologyAdd.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="mFieldsMappingRemoveButton">
       <property name="toolTip">
        <string>Remove the selected or last pair of fields</string>
       </property>
       <property name="icon">
        <iconset resource="../../images/images.qrc">
         <normaloff>:/images/themes/default/symbologyRemove.svg</normaloff>:/images/themes/default/symbologyRemove.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="mFieldsMappingVSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>mIdLineEdit</tabstop>
  <tabstop>mNameLineEdit</tabstop>
  <tabstop>mRelationStrengthComboBox</tabstop>
  <tabstop>mFieldsMappingTable</tabstop>
  <tabstop>mFieldsMappingAddButton</tabstop>
  <tabstop>mFieldsMappingRemoveButton</tabstop>
 </tabstops>
 <resources>
  <include location="../../images/images.qrc"/>
 </resources>
 <connections/>
</ui>
